instance_config.yaml

```diff
@@ -1,40 +1,63 @@
 ---
+## NOTE!
+# NEVER COMMIT A SECRET HERE!
+#
+# This file is meant as a guide, but should not be assumed to deploy to production.
+# To deploy changes to this file to production,
+# change in @edx/veda-secure/shared/instance_config.yaml
+#
+#
 
-# ---
-# AWS
-# ---
-veda_s3_hotstore_bucket:
-veda_deliverable_bucket:
-edx_s3_endpoint_bucket:
-aws_video_images_bucket:
-aws_video_images_prefix: ''
-
-# ---
-# Celery
-# ---
-celery_threads: 1
```

Comment:
@greg Due to removal of this, veda sandbox builds are failing because in [worker.sh](https://github.com/edx/edx-video-worker/blob/master/worker.sh#L14) `clerey_threads` value is expected to be present in `instance_config.yaml`. Should I put it back in this file?